File: app/src/main/java/com/testeandroidv2/repository/response/UserAccountResponse.java
package com.testeandroidv2.repository.response;

import com.google.gson.annotations.SerializedName;

public class UserAccountResponse {

    @SerializedName("userAccount")
    private UserAccount userAccount;

    @SerializedName("error")
    private Error error;

    public UserAccount getUserAccount() {
        return userAccount;
    }

    public void setUserAccount(UserAccount userAccount) {
        this.userAccount = userAccount;
    }

    public Error getError() {
        return error;
    }

    public void setError(Error error) {
        this.error = error;
    }
}

File: App/data/src/main/java/com/santander/data/source/remote/entity/response/UserAccountResponse.kt
package com.santander.data.source.remote.entity.response

import com.google.gson.annotations.SerializedName

data class UserAccountResponse(

	@field:SerializedName("bankAccount")
	val bankAccount: String? = null,

	@field:SerializedName("agency")
	val agency: String? = null,

	@field:SerializedName("balance")
	val balance: Double? = null,

	@field:SerializedName("name")
	val name: String? = null,

	@field:SerializedName("userId")
	val userId: Int? = null
)